hypervisor/Makefile

```diff
@@ -194,7 +194,7 @@ C_SRCS += bsp/$(CONFIG_PLATFORM)/cmdline.c
 else
 ifeq ($(CONFIG_PLATFORM), sbl)
 C_SRCS += boot/sbl/multiboot.c
-C_SRCS += boot/sbl/hob_parse.c
```

Comment:
hob_parse for SBL implementation?

do we need to create a ABL folder? i remembered that you checked this with ACRN team, what's the conclusion then?